Jenkinsfile_k8s

```diff
@@ -173,7 +173,8 @@ pipeline {
         environment {
           // Defines the following environment variables: AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_TENANT_ID, AZURE_SUBSCRIPTION_ID
           // Ref. https://plugins.jenkins.io/azure-credentials/#plugin-content-declarative-pipeline
-          AZURE                         = credentials('packer-azure-serviceprincipal')
+          AZURE                         = credentials('packer-azure-serviceprincipal-sponsorship')
+          AZURE_GALLERY                 = credentials('packer-azure-serviceprincipal')
```

Comment:
```suggestion
```

Removing since we deployed https://github.com/jenkins-infra/azure/pull/560 and https://github.com/jenkins-infra/azure/pull/561 (new galleries in new subscription in us east 2)